zerver/lib/home.py

```diff
@@ -207,6 +208,8 @@ def build_page_params_for_home_page_load(
         # There is no event queue for web_public_visitors since
         # events support for web_public_visitors is not implemented yet.
         no_event_queue=user_profile is None,
+        # Required for about_zulip.hbs
+        zulip_merge_base=ZULIP_MERGE_BASE,
```

Comment:
We should be able to delete the similar entries from the `zerver/views/home.py` context in this commit.

Reply:
There aren't any :)